content/millennium/r4/financial/coverage.md

```diff
@@ -188,15 +203,16 @@ The common [errors] and [OperationOutcomes] may be returned.
 
 ## Patch
```

Comment:
The POST section has some examples corresponding to each scenario. Do you think we need to add example for each scenario here as well or not?

Reply:
POST has not been completed yet. This will come in a future change to fhir.cerner.com, when the POST work for social healthcare is done.

Reply:
POST? That is not available yet, this is just for search and patch on public healthcare.

Reply:
By POST, I meant create operations 
https://fhir.cerner.com/millennium/r4/financial/coverage/#example---patient-level-coverage-1
https://fhir.cerner.com/millennium/r4/financial/coverage/#example---encounter-level-coverage

Reply:
The create operations will not need to be updated for this.

Reply:
No, I mean do we want to update this PATCH operation to add more examples for different scenarios, the way we have examples for different scenarios in CREATE operation

Reply:
Ohhhh, I don't believe so, we support 2 patch operations that are the same as the previous so I don't think we'll need examples for this specifically.